eg/Utils.py

```diff
@@ -394,23 +394,23 @@ def IsVista():
     """
     warnings.warn(
         "eg.Utils.IsVista() is deprecated. "
-        "Use eg.WindowsVersion.IsVista() instead",
+        "Use eg.WindowsVersion >= 'Vista' instead",
```

Comment:
`IsVista()` means *is Vista* and not *is Vista or later*. So i think the message can/should be changed to `"Use eg.WindowsVersion.IsVista() or eg.WindowsVersion >= 'Vista' instead."`